package.json

```diff
@@ -40,7 +40,7 @@
     "node": ">=12"
   },
   "prettier": {
-    "printWidth": 120,
+    "printWidth": 90,
```

Comment:
fun fact, don't use any line length above 80/90 if you plan on easily viewing side by side diffs on github or anywhere really...